saleor/dashboard/product/forms.py

```diff
@@ -300,12 +301,9 @@ class Meta:
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        manager = get_extensions_manager()
+        self.manager = manager = get_extensions_manager()
```

Comment:
I think we can make it simpler:
```suggestion
        self.manager = get_extensions_manager()
```

Reply:
When line length is prefered over simplicity 👀